### ISSUE TITLE
wrap mouse axis x,y

### ISSUE THREAD

**Issue by planer-pro** [moses-palmer/pynput#369]
Hi everyone. Who knows how to do unlimited mouse mooves for x and y axis? Construction like below not work;
`def on_move(x, y):
    if x > 1024:
        mouse.position = (0, y)
    print(x,y)

**Comment by moses-palmer** [comment]
Thank you for your report.

What platform do you use?

If I understand you correctly, when the pointer is past *x = 1024*, your code causes it to jump to the left edge of the screen, only to return to the position past *x = 1024* immediately?

**Comment by planer-pro** [comment]
> Thank you for your report.
> 
> What platform do you use?
> 
> If I understand you correctly, when the pointer is past _x = 1024_, your code causes it to jump to the left edge of the screen, only to return to the position past _x = 1024_ immediately?

yes, you understand right. I use raspberry pi platform. RaspbianOS Linux
